api-reference/timescaledb/continuous-aggregates/create_materialized_view.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: CREATE MATERIALIZED VIEW (Continuous Aggregate)
+sidebarTitle: CREATE MATERIALIZED VIEW
+description: Create a continuous aggregate on a hypertable or another continuous aggregate
+topics: [continuous aggregates]
+keywords: [continuous aggregates, create]
+tags: [materialized view, hypertables]
+license: community
+type: command
+products: [cloud, self_hosted, mst]
+---
+
+import { CAGG, HYPERTABLE, TIMESCALE_DB, PG, CHUNK } from '/snippets/vars.mdx';
+
+<Icon icon="circle-play" iconType="duotone" /> Since 2.22.0
+
+You use the `CREATE MATERIALIZED VIEW` statement to create {CAGG}s. To learn more, see the
+[{CAGG} how-to guides][cagg-how-tos].
+
+The syntax is:
+
+```sql
+CREATE MATERIALIZED VIEW <view_name> [ ( column_name [, ...] ) ]
+  WITH ( timescaledb.continuous [, timescaledb.<option> = <value> ] )
+  AS
+    <select_query>
+  [WITH [NO] DATA]
+```
+
+`<select_query>` is of the form:
+
+```sql
+SELECT <grouping_exprs>, <aggregate_functions>
+    FROM <hypertable or another continuous aggregate>
+[WHERE ... ]
+GROUP BY time_bucket( <const_value>, <partition_col_of_hypertable> ),
+         [ optional grouping exprs>]
+[HAVING ...]
+```
+
+The {CAGG} view defaults to `WITH DATA`. This means that when the
+view is created, it refreshes using all the current data in the underlying
+{HYPERTABLE} or {CAGG}. This occurs once when the view is created.
+If you want the view to be refreshed regularly, you can use a refresh policy. If
+you do not want the view to update when it is first created, use the
+`WITH NO DATA` parameter. For more information, see
+[`refresh_continuous_aggregate`][refresh-cagg].
+
+{CAGG_CAP}s have some limitations of what types of queries they can
```

Comment:
CAGG_CAP variable does not render (it's not imported)

Reply:
Fixed